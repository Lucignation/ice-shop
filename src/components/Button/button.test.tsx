import Button from './button.component';
import renderer from 'react-test-renderer';

const mockProps = {
  btnText: 'mock BtnText',
};
describe('Button : ', () => {
  it('renders a snapshot of <Button /> when btnType is primary ', () => {
    const button = renderer
      .create(<Button btnType='primary' {...mockProps} />)
      .toJSON();
    expect(button).toMatchSnapshot();
  });


    it('renders a snapshot of <Button /> when btnType is something other primary ', () => {
      const button = renderer
        .create(<Button {...mockProps} />)
        .toJSON();
      expect(button).toMatchSnapshot();
    });
});
