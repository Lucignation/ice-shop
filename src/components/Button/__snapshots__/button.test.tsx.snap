// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button :  renders a snapshot of <Button /> when btnType is primary  1`] = `
<div>
  <input
    className="submit_btn"
    type="submit"
    value="mock BtnText"
  />
</div>
`;

exports[`Button :  renders a snapshot of <Button /> when btnType is something other primary  1`] = `
<div>
  <input
    className="submit_btn_secondary"
    type="submit"
    value="mock BtnText"
  />
</div>
`;
